### PR TITLE
arm64: dts: qcom: msm8916-lg-m216: Add bms and charging

### PR DESCRIPTION
**Pull request by cristianc**
arm64: dts: qcom: msm8916-lg-m216: Add bms and charging

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/msm8916-lg-m216.dts [hunk 1]
```diff
@@ -40,6 +40,24 @@
 		};
 	};
 
+	bat: battery {
+		compatible = "simple-battery";
+		voltage-min-design-microvolt = <3300000>;
+		voltage-max-design-microvolt = <4350000>;
+		#energy-full-design-microwatt-hours = <9500000>;
```
Comment (TravMurav) [inline arch/arm64/boot/dts/qcom/msm8916-lg-m216.dts:47]: Not sure why # is there

**Review by TravMurav — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/qcom/msm8916-lg-m216.dts (hunk 1 above)
Comment (cristianc) [reply arch/arm64/boot/dts/qcom/msm8916-lg-m216.dts:47]: I read "Typ: 2300mAh / 8.8Wh  Min: 2220mAh / 8.5Wh" on BL-45A1H battery back side. I'm not sure what is the right value for energy-full-design-microwatt-hours property. You said it's just informational, though.

Comment (TravMurav) [reply arch/arm64/boot/dts/qcom/msm8916-lg-m216.dts:47]: I mean `#` symbol but if it says 8.8Wh typical - that's what you need

**Review by TravMurav — COMMENTED** [review]
Please drop second line in the commit message and sign it off

**Review by TravMurav — COMMENTED** [review]
Upstream requires real real name in a signoff so we do the same.
See this for more info:
https://github.com/msm8916-mainline/linux/blob/15593c996c2b49e931c6edb9516847ad199d648e/CONTRIBUTING.md

You may want to reconfigure your git config either globally or just for this repository since Author name should also match

**Pull request closed without merging**